docs/apache-airflow/authoring-and-scheduling/timetable.rst

```diff
@@ -217,3 +228,24 @@ is immediately triggered after you re-enable the DAG.
 
 By these examples, you see how `CronTriggerTimetable`_ triggers DAG runs is more intuitive and more similar to what
 people expect cron to behave than how `CronDataIntervalTimetable`_ does.
+
+
+.. _Differences between the two data intervals timetables:
+
+Differences between the two data intervals timetables:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As for the wo cron timetables, there is no difference between both of the data interval timetables when ``catchup`` is ``True``.
```

Comment:
They are still different if the `start_time` does not align with the cron expression. Say you have `start_date=datetime(2023, 1, 1, 1)`, a daily cron runs from midnight to midnight (starting from 2nd Jan), while a `timedelta(days=1)` schedule would run everyday at 1am (starting from 1st Jan).